src/Compilation.zig

```diff
@@ -27,6 +38,8 @@ system_include_dirs: std.ArrayList([]const u8),
 output_name: ?[]const u8 = null,
 builtin_header_path: ?[]u8 = null,
 target: std.Target = @import("builtin").target,
+mimic: MimicCompiler = .clang,
```

Comment:
There's an `emulate` field in `LangOpts` that was recently added that does the same thing. We could also move the `dialect` there.

Reply:
I switched `mimic` to use `LangOpts` and deleted `dialect`.
I'll add `dialect` back in when I get there. I think I got a little ahead of myself. When I started this, I thought it was going to be a lot easier than it's turned out to be :)